### PR TITLE
B dev

### PR DESCRIPTION
**Pull request by betiniakarandut**
1. Merged series of conflicts
2. Feat: Implemented the ProfileDropDownMenu for talent avatar

### CODE REVIEW

**Review by Ayobami6 — CHANGES_REQUESTED** [review]
Bro, This is a lot of conflict, please pull first from staging to local branch, then fix the conflicts you might, after then make another PR from your updated branch

File: server/controllers/offerController.ts
Comment (Ayobami6) [inline server/controllers/offerController.ts:1]: It looks like you have 2 getOfferByTalent and getOfferByEmployer methods here,  So it's not allowing the application build or bundle

Comment (betiniakarandut) [reply server/controllers/offerController.ts:1]: Duplicates removed. It is running smooth on my end now.

**Review by Ayobami6 — APPROVED** [review]
Alright thanks, I will fix the test from here

**Pull request merged**